CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ All notable changes to [zeebe-bpmn-moddle](https://github.com/zeebe-io/zeebe-bpm
 
 ___Note:__ Yet to be released changes appear here._
 
+## 0.8.0
+* `FEAT`: make Message Intermediate Throw Event and Message End Event Zeebe Service Tasks ([#15](https://github.com/zeebe-io/zeebe-bpmn-moddle/pull/15))
```

Comment:
How about if we mention Zeebe 1.2 here? I like how we stated this in https://github.com/zeebe-io/zeebe-bpmn-moddle/pull/16/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL15

Reply:
Makes sense. Added the '_to support Zeebe 1.2_' part